stl/inc/forward_list

```diff
@@ -241,9 +241,10 @@ template <class _Value_type, class _Voidptr>
 struct _Flist_node { // forward_list node
     using _Nodeptr = _Rebind_pointer_t<_Voidptr, _Flist_node>;
 
-    _Nodeptr _Next; // successor node
-    _Value_type _Myval; // the stored value
+    _Nodeptr _Next;
+    _Value_type _Myval = _Returns_exactly<_Value_type>(); // fake a viable constructor to workaround GH-2749

```

Comment:
Good catch! It works because we have two of them! :joy_cat:

The first declaration is guarded by `#if _HAS_CXX20`:
https://github.com/microsoft/STL/blob/7f04137880e1c3df5125c1baf808f16f399dee2e/stl/inc/type_traits#L1221-L1226

The second declaration is unconditional:
https://github.com/microsoft/STL/blob/7f04137880e1c3df5125c1baf808f16f399dee2e/stl/inc/type_traits#L1609-L1618

I will validate and push changes. (The code isn't wrong, but we should avoid such redundant declarations, especially because there is an active modules bug related to them.)